docs/platforms/unity/tracing/instrumentation/custom-instrumentation.mdx

```diff
@@ -15,3 +15,5 @@ To capture transactions and spans customized to your organization's needs, you m
 <PlatformContent includePath="performance/add-spans-example" />
 
 <PlatformContent includePath="performance/retrieve-transaction" />
+
+<PlatformContent includePath="performance/improving-data" />
```

Comment:
suggestion: Would you mind making equivalent changes to the `dotnet` docs as well?

I really like this documentation. And I'd love to have it on the .NET SDK documentation too.